.github/workflows/upgrade-python-requirements.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: setup testeng-ci
         run: |
-          git clone https://github.com/edx/testeng-ci.git
+          git clone https://github.com/openedx/testeng-ci.git
```

Comment:
This repo did not get moved as a part of the transition and still lives in the `edx` org. We should not update this URL.

Reply:
Thanks - I'm going to address this after I set up the automation to correctly update pylintrc

Reply:
The correct solution is to use the `.github/workflow-templates/upgrade-python-requirements.yml` file, which I've done now.